python/seldon_deploy_sdk/auth/openid.py

```diff
@@ -109,6 +107,8 @@ def _use_authorization_code(self):
             state=self._AuthCodeState,
             scope=self._config.scope,
         )["url"]
+
+        webbrowser.open_new_tab(request_url)
```

Comment:
:question: Should we have _either_ a webbrowser tab opening _or_ printing the below?  Otherwise it's potentially a little confusing to users to be told to do something that's already happening?

Reply:
I see what you mean. 

I think it makes sense to give the same message twice. Although I'll rephrase the prompt to acknowledge that the URL should have had already opened on a new tab.